kwanghyun/services/likeService.js

```diff
@@ -0,0 +1,11 @@
+const likeDao = require("../models/likeDao");
+
+const likes = async (userId, postId) => {
+  const likes = await likeDao.likes(userId, postId);
+
+  return likes;
```

Comment:
```suggestion
  return likeDao.likes(userId, postId);
```
굳이 변수 선언 및 동기처리를 통해 두 줄로 작성될 필요가 없는 코드입니다.